thirdparties-extension/fr.opensagres.poi.xwpf.converter.pdf.itext5/src/test/java/org/apache/poi/xwpf/converter/pdf/PdfConverterTestCase.java

```diff
@@ -47,6 +48,7 @@ protected void doGenerate( String fileInName )
 
         long startTime = System.currentTimeMillis();
 
+        ZipSecureFile.setMinInflateRatio(0);
```

Comment:
Could you add a comment please to explain why you must do that?

Reply:
From what I gathered the documents used in the tests are very compressed, and POI has a zip bomb detection mechanism, so this is in order to avoid the false positives, now that I think about it I will move that into the tests. PR will be updated in a moment

Reply:
Nevermind, its already on the test, this is what I get from replying from my phone. Will comment

Reply:
> now that I think about it I will move that into the tests

What about add an option to manage that?

Reply:
An option to manage does not make much sense to me since it is static, adding an option will make it less evident that this will affect everything that depends on ZipSecureFile. I would rather leave it as is. If anyone using this library faces this limit the error message is pretty explicit:

> java.io.IOException: Zip bomb detected! The file would exceed the max. ratio of compressed file size to the size of the expanded data.
> This may indicate that the file is used to inflate memory usage and thus could pose a security risk.
> You can adjust this limit via ZipSecureFile.setMinInflateRatio() if you need to work with files which exceed this limit.
> Uncompressed size: 124931, Raw/compressed size: 1243, ratio: 0.009949

The default value for the ratio (0.01) is also pretty close to the one being detected on the tests.

Reply:
Agree with you, thanks for your clarification.